### PR TITLE
SOLR-15786

### PR DESCRIPTION
**Pull request by epugh**
https://issues.apache.org/jira/browse/SOLR-15786
# Description

Integrate "films" into the list of example collections via `bin/solr start -e` command.

# Solution

This is a babystep towards potentially shipping Solr with more examples that could demonstrate various features...   I'm thinking of a future where you fire up the example of Solr that demos some of the streaming expressions that are in the Ref Guide ;-)

# Tests

Manually tested with regular and solr cloud modes.

# Checklist

Please review the following and check all that apply:

- [X ] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ X] I have created a Jira issue and added the issue ID to my pull request title.
- [ X] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [ X] I have developed this patch against the `main` branch.
- [ X] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

**Comment by epugh** [comment]
@erikhatcher you want to review?

**Comment by epugh** [comment]
@janhoy I'd love to have you look at this?!

File: solr/core/src/java/org/apache/solr/util/SolrCLI.java [hunk 1]
```diff
@@ -2845,8 +2845,42 @@ protected void runExample(CommandLine cli, String exampleName) throws Exception
           echo("exampledocs directory not found, skipping indexing step for the techproducts example");
         }
       }
+      else if ("films".equals(exampleName) && !alreadyExists) {
+        echo("Adding name and initial_release_data fields to films schema \"_default\"");
 
-      echo("\nSolr "+exampleName+" example launched successfully. Direct your Web browser to "+solrUrl+" to visit the Solr Admin UI");
+        HttpSolrClient solrClient = new HttpSolrClient.Builder(solrUrl).build();
+        try {
+          SolrCLI.postJsonToSolr(solrClient, "/" + collectionName + "/schema", "{\n" +
```
Comment (HoustonPutman) [inline solr/core/src/java/org/apache/solr/util/SolrCLI.java:2853]: > Example of using the _default configset and adding fields dynamically.

That makes me think that films is just using the dynamic fields available in _default, but here you are adding a few fields. Maybe we convert it to only use dynamic fields, or change the description to be a little clearer.

**Review by HoustonPutman — COMMENTED** [review]
Looks good to me, though I haven't run through the example yet.

This may be a separate concern, but I think it would be really good to have an example of a collection that just uses dynamic fields provided with the default configset, not adding any additional fields. Since this is a use case many users go with

File: solr/core/src/java/org/apache/solr/util/SolrCLI.java (hunk 1 above)
Comment (epugh) [reply solr/core/src/java/org/apache/solr/util/SolrCLI.java:2853]: Took another stab at this!   If you think using _default is just a bad idea/not helpful, would you mind opening another JIRA and I'll do the migration there ;-)????

Comment (epugh) [reply solr/core/src/java/org/apache/solr/util/SolrCLI.java:2867]: Reading through https://github.com/apache/solr/tree/main/solr/example/films, the whole example makes a lot of assumptions that you are using _default.   If we get rid of schema autoguessing, then this whole example provides much less value.    (Oh, and despite it being an example of the `useParams` parameter according to ref guide, I don't actually see it mentioned or used!  sigh)

Comment (epugh) [reply solr/core/src/java/org/apache/solr/util/SolrCLI.java:2867]: I ended up digging into the README for the films example and created https://issues.apache.org/jira/browse/SOLR-15834

**Pull request merged**